docs/src/content/en/docs/community/contributing-templates.mdx

```diff
@@ -0,0 +1,192 @@
+- **GitHub URL** - Link to your template repository
+- **Description** - Detailed explanation of what the template does and its value
+- **Optional Image** - Screenshot or diagram showing the template in action
+- **Optional Demo Video** - Link to a video demonstrating the template's functionality
+
+## Review Process
+
+### Review Criteria
+
+Templates are evaluated on:
+
+- **Technical compliance** - Adherence to template rules and conventions
+- **Code quality** - Clean, maintainable, and well-documented code
+- **Uniqueness** - Novel use cases or innovative implementation patterns
+- **Educational value** - Ability to teach Mastra concepts effectively
+- **Community benefit** - Potential value to the broader Mastra community
+
+### Feedback and Iteration
+
+If your template needs improvements:
+
+- You'll receive specific feedback on required changes
+- Make the requested modifications and resubmit
+- The review process continues until the template meets standards
+
+## Community Guidelines
+
+### Template Ideas
+
+Consider creating templates for:
+
+- **Industry-specific use cases** - Healthcare, finance, education, etc.
+- **Integration patterns** - Specific API or service integrations
+- **Advanced techniques** - Complex workflows, multi-agent systems, or novel patterns
+- **Learning resources** - Step-by-step tutorials for specific concepts
+
+### Development Best Practices
+
+- **Start simple** - Begin with a minimal working example and add complexity gradually
+- **Document thoroughly** - Include detailed comments and comprehensive README
+- **Test extensively** - Verify your template works across different environments
+- **Seek feedback** - Share with the community for early feedback before submission
+
+### Community Engagement
+
+- **Join Discord** - Participate in the [Mastra Discord community](https://discord.gg/BTYqqHKUrf)
+- **Share progress** - Update the community on your template development
+- **Help others** - Assist other contributors with their templates
+- **Stay updated** - Keep track of new Mastra features and conventions
+
+## Template Maintenance
+
+### Ongoing Responsibilities
+
+As a template contributor, you may be asked to:
+
+- **Update dependencies** - Keep templates current with latest Mastra versions
+- **Fix issues** - Address bugs or compatibility problems
+- **Improve documentation** - Enhance instructions based on user feedback
+- **Add features** - Extend templates with new capabilities
+
+### Community Support
+
+The Mastra team and community provide:
+
+- **Technical guidance** - Help with complex implementation challenges
+- **Review feedback** - Detailed feedback to improve template quality
+- **Promotion** - Showcase approved templates to the community
+- **Maintenance assistance** - Support for keeping templates up-to-date
+
+## Validation Checklist
+
+Before submitting a template, verify:
+
+- [ ] All code organized in `src/mastra/` directory
```

Comment:
What's the idea with formatting here? `[ ]` looks a little strange, is there a reason for it?